search.json

```diff
@@ -8,7 +8,8 @@ layout: null
       "category" : "{{ doc.category }}",
       "tags"     : "{{ doc.tags | join: ', ' }}",
       "url"      : "{{ site.baseurl }}{{ doc.url }}",
-      "date"     : "{{ doc.date }}"
+      "date"     : "{{ doc.date }}",
+      "body"     : "{{ page.content | strip_html | strip_newlines | replace: '\', '\\\\' | replace: '"', '\\"' | replace: '	', '    ' }}"
```

Comment:
```
replace: '\', '\\\\'
```
This code means that the escape character `\` that will destroy the json structure converts to `\\`.
Because in ruby ​​strings, `\` is also an escape character, so it needs to be written as `\\\\`.